urdf/ur.urdf.xacro

```diff
@@ -17,6 +17,10 @@
    <xacro:arg name="safety_k_position" default="20"/>
    <!-- ros2_control related parameters -->
    <xacro:arg name="headless_mode" default="false" />
+   <xacro:arg name="robot_ip" default="0.0.0.0" />
+   <xacro:arg name="script_filename" default=""/>
```

Comment:
I would like to see here some default values. Otherwise, this has to be defined even for a default use-case.

Reply:
Hm, having these like that shouldn't be a problem until you actually want to create a driver. And specifying the robot_ip and script_filename there seems very reasonable. We had the same before, btw. This block gets removed by this PR from [urdf/ur_macro.xacro](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/pull/21/files#diff-ce4e0c83b6516d2103ff61d05a6a19cf6198e28c7c97913da4d69b56269cfd44):


```xml
    <!-- Data files required by the UR driver -->
    <xacro:arg name="script_filename" default=""/>
    <xacro:arg name="output_recipe_filename" default=""/>
    <xacro:arg name="input_recipe_filename" default=""/>
    <xacro:arg name="robot_ip" default="10.0.1.186"/>
```

Reply:
you are right!